cli/src/plz/cli/controller_proxy.py

```diff
@@ -246,3 +252,12 @@ def harvest(self) -> None:
 def _check_status(response: requests.Response, expected_status: int):
     if response.status_code != expected_status:
         raise RequestException(response)
+
+
+def _read_response_in_chunks(http_response: Response) \
+        -> Iterator[bytes]:
+    while True:
+        bs = http_response.raw.read(1024 * 1024)
```

Comment:
Should this refer to `_HTTP_RESPONSE_READ_CHUNK_SIZE`? Or are they the same value by coincidence?

Reply:
Heheh, next time you wonder why I ask you to review the code even if I know the codebase better, please remember this time :)